src/services/pool/pool.service.ts

```diff
@@ -132,12 +132,6 @@ export default class PoolService {
     return (this.pool.linearPoolTokensMap = linearPoolTokensMap);
   }
 
-  removeBptFromTokens(): string[] {
-    return (this.pool.tokensList = this.pool.tokensList.filter(
-      address => !isSameAddress(address, this.pool.address)
-    ));
-  }
```

Comment:
Are you sure this isn't used anywhere? Search for `.removeBptFromTokens()`

Reply:
yes, now it is not used anywhere